pyproject.toml

```diff
@@ -6,8 +6,8 @@ build-backend = "setuptools.build_meta"
 name = "argilla"
 description = "Open-source tool for exploring, labeling, and monitoring data for NLP projects."
 readme = "README.md"
-requires-python = ">=3.7"
-license = {text = "Apache-2.0"}
+requires-python = ">=3.7"  # TODO: upgrade minimum Python version to 3.8, and pin maximum to < 3.12
```

Comment:
We will deprecate `3.7` in the `1.9.0` release, right?

Reply:
Yes I think so, didn't want to update that yet!